src/main/kotlin/no/skatteetaten/aurora/cantus/controller/DockerRegistryControllerOld.kt

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
Skal denne være kommentert ut?

Reply:
reverter for å bevare gammel kompatibilitet: fikset